e2e/test/MessageSendE2ETests.cs

```diff
@@ -157,6 +158,21 @@ public async Task X509_DeviceSendSingleMessage_Http()
             await SendSingleMessage(TestDeviceType.X509, Client.TransportType.Http1).ConfigureAwait(false);
         }
 
+        public async static Task SendMultipleMessages(TestDeviceType type, Client.TransportType transport)
+        {
+            TestDevice testDevice = await TestDevice.GetTestDeviceAsync(DevicePrefix, type).ConfigureAwait(false);
+
+            using (DeviceClient deviceClient = testDevice.CreateDeviceClient(transport))
+            {
+                await deviceClient.OpenAsync().ConfigureAwait(false);
+                for(int i = 0; i < 50; i++)
```

Comment:
The value 50 should be moved to a constant and a comment should be added to clarify why it is 50.

Reply:
Sure will add comment